lib/entities/release.ts

```diff
@@ -80,12 +81,24 @@ export type ReleaseSysProps = {
   updatedAt: ISO8601Timestamp
   lastAction?: Link<'ReleaseAction'>
 }
-
+export type ReleasesReferenceFilters = ScheduledActionReferenceFilters
+
+export type MetadataAggeratedRootProps = {
+  metadata: {
+    withReferences: [
+      {
+        entity: Link<'Entry'>
+        filter: Record<ScheduledActionReferenceFilters, string[]>[]
```

Comment:
In order to not reference scheduled actions here, could you add an alias in `lib/common-types.ts` like this:

```
export type ReleasesReferenceFilters = ScheduledActionsReferenceFilters
```
and then import `ReleaseReferenceFilters` here?